tests/fixtures/addons_info.json

```diff
@@ -121,6 +121,8 @@
     "version": "9.14.0",
     "update_available": false,
     "watchdog": false,
-    "devices": []
+    "devices": [],
+    "system_managed": false,
+    "system_managed_config_entry": null
```

Comment:
Side note: It would be good to exercise all the optional parameters with an argument that is not null with additional test fixtures and parametrized tests.

Reply:
Ok yea Ill put that on the todo list, makes sense